taxi/admin.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Manufacturer, Driver
+
+
+admin.site.register(Manufacturer)
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = ["manufacturer",]
```

Comment:
> Make it possible to filter Car by manufacturer.

Right now filter doesn't appear. You need to specify fields on a link to manufacturer to make it work